dbcog/find_monster/tokens/find_monster_tokens.py

```diff
@@ -474,6 +523,30 @@ async def string_to_token(string: str, dbcog: "DBCog") -> QueryToken:
     return RegularToken(string, negated=negated, exact=exact)
 
 
+async def prune_results_subquery(monster_info: "MonsterInfo") -> "MonsterInfo":
+    exacts = set()
+    for monster in monster_info.valid_monsters:
+        matched = False
+        for token in monster_info.monster_matches[monster].name:
+            if token.match_data.name_type != "fluff":
```

Comment:
we should make this an enum (like, don't compare to a string value) but we can do that later